fleaapi/controllers/forget_test.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import User
```

Comment:
The implementation is using `User` object from our models but the tests are running against Django's builtin User model. I don't think the test will succeed here. Correct me if I'm wrong.